### ISSUE TITLE
ConnectException for files_external:notify

### ISSUE THREAD

**Issue by MorrisJobke** [nextcloud/server#2527]
Running `files_external:notify` with username and password for user shares that use User credentials cause a ConnectException:

```
$ occ files_external:notify --user abc --password abc 4

[Icewind\SMB\Exception\ConnectException]
Unknown error (stdbuf: unrecognized option '--authentication-file=/proc/self/fd/3'
Try 'stdbuf --help' for more information.)
```

cc @icewind1991

**Comment by icewind1991** [comment]
make sure `sbmclient` (the binary) is installed and in path

**Comment by MorrisJobke** [comment]
> make sure sbmclient (the binary) is installed and in path

Okay. Task for you: Detect this and show a proper error message ;)

**Comment by icewind1991** [comment]
https://github.com/nextcloud/server/pull/2528

**Comment by MorrisJobke** [comment]
Fixed by #2676 in Nextcloud 12.
